cartoframes/assets/vector.html

```diff
@@ -61,7 +65,7 @@
           new carto.Viz(elem['styling'])
       );
       var last_source = idx === 0 ? 'watername_ocean' : 'layer' + (idx - 1);
-      temp.addTo(map, last_source);
+      temp.addTo(map);
```

Comment:
`last_source` needs to be returned here. The old code had:
```js
temp.addTo(map, last_source)
```
There may be a better name than `watername_ocean`, or we should detect if it's a CARTO basemap -- and use `watername_ocean` if so, otherwise just plop it on the top.